src/EllCrv/EllCrv.jl

```diff
@@ -712,38 +703,125 @@ end
 #  Division polynomials
 #
 ################################################################################
-# TODO:
-# - Adjust to Sage/Magma
-# - Implement for long form, see Mazur, Tate, "p-adic sigma function"
-
+"""
+Computes the n-th division polynomial of an elliptic curve defined over a field k following Mazur and Tate. The two-torsion multiplicity flag determines the output. 
```

Comment:
The docstrings need some fixes, see https://docs.julialang.org/en/v1/manual/documentation/.

1. Should be imperative ("compute" instead of "computes"). I know we are inconsistent, but we are trying to adjust :)
2. The first line should look like
```
    division_polynomial(E::EllCrv, n::Int, two_torsion_multiplicity::Int, x, y) -> Poly
```
3. Lines should not be longer than 80 characters.
```